p4_symbolic/ir/ir.proto

```diff
@@ -100,6 +109,33 @@ message TableImplementation {
 
   // Table type with respect to action selection.
   ActionSelectorType action_selector_type = 1;
+
+  // Default action name (e.g. when no other entry is matched).
+  string default_action = 2;
+
+  // Hexstring values passed to default action as parameters.
+  repeated string default_action_parameters = 3;
+
+  // Maps every action used by this table (by name) to the next control block
+  // to be taken (also by name) if that action was matched.
+  // if an action is not in this map, then that action is the end of the flow.
+  map<string, string> next_flows = 4;
```

Comment:
Since this is a string -> string map and we don't have any type safety, could we give this a very explicit name? For example, action_to_next_control_map?

Reply:
I want to use the word flow because it is used in other places in the IR, code, and in documentation I saw online.

How about action_to_next_flow? (map is redundant with the type)

Reply:
I agree we don't need _map.
"Flow" is a term that is overloaded to an absurd extend at Google and in networking in general; it means a lot of things including table entry, set of packets with similar headers, a data structure used in OpenFlow, etc. I would be very happy if we could avoid to give it yet another meaning here.

Reply:
I am persuaded, let me fix this

Reply:
done